### DISCUSSION TITLE
[Feature] Rename photos/videos and add descriptions

### DISCUSSION THREAD

**Discussion by Flaxee** [immich-app/immich#1775] [Feature Request]
### The feature

I'd appreciate the possibility to rename uploaded photos/videos and being able to adding a description in the WebUI/mobile app. Since Immich has read-write support for every file, this shouldn't be a problem.
This would also make it easier to search for specific photos later on, when the search function is working.

As of right now, you'd have to make sure the photos are named correctly before uploading them. This is fine most of the time I guess, but it would be cool if you could rename them (or at least add a description) through Immich.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by madkatz01** [comment]
+1, I agree this would be a nice feature

**Reply by jrasm91** [reply]
If you want this feature, you can hit the up-vote arrow on it to vote for it :smile:

**Comment by jrasm91** [comment]
Description was added in #2237.

We can use a separate issue to track editing filename and/or other metadata if needed.

**Reply by Gifford47** [reply]
Is there already a new issue for that feature? I would also appreciate it too!

**Reply by jrasm91** [reply]
Take a look at https://github.com/immich-app/immich/discussions/1696
